tools/cldr-apps/js/src/views/GeneralInfo.vue

```diff
@@ -0,0 +1,74 @@
+<!--
+    This replaces cldrLoad.loadGeneral(itemLoadInfo);
+    Note that there isn't any provision for showPossibleProblems, which seems to be
+    entirely dead code.
+-->
```

Comment:
this comment maybe belongs in a commit message rather than in the code itself, since it refers to code that's being removed -- that could be in a follow-up PR